### ISSUE TITLE
Feature Suggestion: Historical Log of Deploys per Deployment

### ISSUE THREAD

**Issue by paolaperaza** [astronomer/astronomer#300]
**Use Case**
Suggested by a customer. As an Astronomer User who deploys often to a variety of deployments, our customers might want to see a historical log of all deploys to each of their deployments.

**Potential Solution**
This is tangentially related to [this issue](https://github.com/astronomer/houston-api/issues/48) (Ability to send Slack notifications on successful deployments).

In addition to sending Slack notifications on successful deployments, a customer might expect to see information on deploys in the Astronomer UI on `/deployments/{deployment_name}` alongside `Overview`, `Configure`. This might include a timestamp, success/failed status, deploy tag, the user that pushed the deploy, etc.

Perhaps this is something a user could export/route elsewhere?

**Additional context**
Customer: "If such a thing didn’t have a place in astronomer itself, some kind of webhook would at least allow anyone to themselves capture and persist the log of what tags where deployed when."

Additional/separate question: If a deployment is deprovisioned, would this historical log go with it? Would customers ever want a history of all deployments they've ever had in their workspace, and a corresponding historical log of deploys alongside it?

@schnie I'm throwing this in this repo, but let me know if you think it belongs elsewhere/if it overlaps with anything else we're already working on or have in mind.

**Comment by schnie** [comment]
Yep, totally makes sense and great feature. We also have a slightly broader issue somewhere (I think?), that describes an audit log, which is an even broader version of this and might include some other events.

**Comment by paolaperaza** [comment]
@schnie Coo! Found that other issue: https://github.com/astronomer/astronomer/issues/228 (Audit Logs).

It seems like an audit or changelog in Orbit for all events in a workspace (user added to a workspace, deployment created, credit card added), etc. might encompass everything in here too, but I'm thinking we can keep this specific issue up for deployment-level logs and we can build from there.

**Comment by schnie** [comment]
Sounds like a plan!
